plugins/node/opentelemetry-instrumentation-ioredis/src/types.ts

```diff
@@ -40,6 +40,16 @@ export type DbStatementSerializer = (
   cmdArgs: IORedisCommand['args']
 ) => string;
 
+export interface IoRedisRequestHookInformation {
```

Comment:
Shouldn't this be 
```suggestion
export interface IORedisRequestHookInformation {
```
as everywhere you have `IORedis....`